ionic_user_interface/src/views/ViewRoute.vue

```diff
@@ -99,8 +104,8 @@
             </ion-card-header>
             <ion-card-content class="ion-no-padding ion-no-margin display-flex">
```

Comment:
`Report` button and `delete comment` button no longer working

Reply:
oops my bad. I realize I made a change such that you can't apply any @click or :hover onto an ion-icon, always need to wrap in a div/other container. Fixed by wrapping in a div.